client/web/compose/src/components/PageBlocks/IFrameConfigurator.vue

```diff
@@ -2,10 +2,11 @@
   <b-tab :title="$t('iframe.label')">
     <b-form-group
       v-if="enableFromRecordURL"
-      class="form-group"
       :label="$t('iframe.srcFieldLabel')"
       :description="$t('iframe.srcFieldDesc')"
       :disabled="!fields.length"
+      label-class="text-primary"
+      class="form-group"
```

Comment:
i dont think we need the class="form-group" if the element is a <b-form-group>
Same in other places